SeaPkg/Tools/GenSeaArtifacts/gen_aux/src/util.rs

```diff
@@ -1,4 +1,11 @@
 //! A module containing utility functions for extracting information from the PDB file.
+//! 
+//! ## License
+//!
+//! Copyright (C) Microsoft Corporation. All rights reserved.

```

Comment:
```suggestion
//! Copyright (c) Microsoft Corporation.
```

Reply:
Gah. that will be all over the place. I'll fix.

Reply:
Updated